pkg/network/dns/parser.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/google/gopacket"
 	"github.com/google/gopacket/layers"
 	"github.com/pkg/errors"
+	stringintern "go4.org/intern"
```

Comment:
Technically this library can intern any comparable value, so aliasing to `stringintern` is misleading IMHO.